<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>213</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>544</width>
     <height>183</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>20</number>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_username">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
      <property name="buddy">
       <cstring>edit_username</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit_username">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;登录用户名&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_password">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
      <property name="buddy">
       <cstring>edit_password</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_password">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;登录密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="button_login">
        <property name="font">
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_register">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_exit">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请输入用户名与密码，并点击登录，否则请注册&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_status">
      <property name="text">
       <string>正在连接数据库...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edit_username</tabstop>
  <tabstop>edit_password</tabstop>
  <tabstop>button_login</tabstop>
  <tabstop>button_register</tabstop>
  <tabstop>button_exit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_exit</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
